Code/GraphMol/TautomerQuery/TautomerQuery.h

```diff
@@ -91,7 +104,54 @@ class RDKIT_TAUTOMERQUERY_EXPORT TautomerQuery {
 
   const std::vector<size_t> getModifiedBonds() const { return d_modifiedBonds; }
 
+  //! serializes (pickles) to a stream
+  void toStream(std::ostream &ss) const;
+  //! returns a string with a serialized (pickled) representation
+  std::string serialize() const;
+  //! initializes from a stream pickle
+  void initFromStream(std::istream &ss);
+  //! initializes from a string pickle
+  void initFromString(const std::string &text);
+
   friend class TautomerQueryMatcher;
+
+#ifdef RDK_USE_BOOST_SERIALIZATION
+  template <class Archive>
+  void save(Archive &ar, const unsigned int version) const {
+    RDUNUSED_PARAM(version);
```

Comment:
Isn't the new style to not use RDUNUSED_PARAM?

Reply:
since it's in the header here, I think it makes sense to keep the argument name there as documentation